bin/hwctl

```diff
@@ -194,13 +194,24 @@ function storage {
 				exit 1
 			fi
 
-			if [ ! -d "$args" ]; then
-				echo "The specified path ($args) does not exist or is not a directory"
+			if [ -z $args ]; then
+				echo "Please specify a valid partition or mounted partition path"
+				exit 1
+			fi
+
+			args_clean=$args
+			if [ "$args" != "/" ]; then
```

Comment:
The case of `/` still fails but only on Fedora Silverblue (works on Fedora Workstation). In rpm-ostree land the persistent mount is at `/var` and as far as `df` is concerned, `/` is not a mount from a physical/"real" partition due to the read-only layout.

```
+ args=/
+ '[' -z / ']'
+ args_clean=/
+ '[' / '!=' / ']'
++ df -h --output=source,target
++ grep '/\(\s\|$\)'
++ tail -n 1
++ cut '-d ' -f2-
++ tr -s ' '
+ mounted_path=
+ '[' '!' -d '' ']'
+ echo '/: Not a valid partition or mounted partition path'
/: Not a valid partition or mounted partition path
```

Here is some example output. `lsblk` reports the information more accurately but it looks harder to parse.

```
$ df -h --output=source,target 
Filesystem     Mounted on
/dev/sda3      /sysroot
devtmpfs       /dev
tmpfs          /dev/shm
tmpfs          /run
tmpfs          /tmp
/dev/sda3      /var
/dev/sda2      /boot
/dev/sda3      /var/home
tmpfs          /run/user/1000
$ lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
sr0     11:0    1 1024M  0 rom  
zram0  251:0    0    8G  0 disk [SWAP]
sda    252:0    0   60G  0 disk 
├─sda1 252:1    0    1M  0 part 
├─sda2 252:2    0    1G  0 part /boot
└─sda3 252:3    0   59G  0 part /var/home
                                /var
                                /sysroot/ostree/deploy/fedora/var
                                /usr
                                /etc
                                /
                                /sysroot
```

Reply:
Based on a side conversation we had, this is expected behavior since `/` is not a real mount and is read-only. We can go ahead and merge this then.